README.md

```diff
@@ -38,9 +38,12 @@ sudo apt-get install x11-xserver-utils
 sudo apt install git
 
 ```
+
 3. Install Dependencies
 
-```
+```sh
```

Comment:
Hello can you tell me the significance of adding the 'sh' right here? I feel like I'm getting something wrong in the install process.

Reply:
This is just for the formatting/colors of the code block and should not affect any text you would paste into your terminal. Here is an example for a python code block where I added `python` instead of `sh`:

``` python
def test():
    print("this should appear in different colors")
```

As far as I see it only highlights the `cd` and therefore could be removed since it does not help readability as much as I anticipated.

Reply:
Ok, I was taking that literally and typing 'sh' before I installed things. It would remove my 'pi@raspberrypi$:' and just replace it just a '$:'. I am sure that messed up my install somehow. Thanks for clearing that up for me.

Reply:
Did it work after you tried it again without the `sh` command?

I am currently getting a blank (openbox?) screen after running all the steps but haven’t had the time to investigate why it doesn’t start the python script and doesn’t show the gui.

Reply:
I am still reinstalling everything and setting everything back up for the 7th time so I will report back when I get it all going. In my past trials I've found that it will boot to the openbox(black) screen and then not run the gui if there are errors somewhere. I left the '-- -no cursor' out of the .bash_profile file to get a cursor where I could then right click on the blank screen, open a terminal, try and run spotifypod.py again and see what errors it gave me.

Reply:
Ok, so I installed everything over without 'sh' got to the point where i had to rightclick on the black openbox screen, open a terminal, ran python3 spotifypod.py.  Spotifypod fetched all of my spotify information and then kicked out this error:

'ImportError: libopenjp2.so.7: cannot open shared object file: No such file or directory'

I remember seeing in one of the issues threads that someone recommended installing something like libopenjp2 but i cant remember where that was.  I am going to have a look again.

Reply:
I found the comment on one of the older commits.  @rsappia mentions not to forget to include apt-get install libopenjp2-7. I am going to install that and see if takes care of my problem.